ElevatorSimulator/Elevator.cs

```diff
@@ -0,0 +1,161 @@
+﻿using ElevatorSimulator.CustomeComponents;
+using System;
+using System.Linq;
+using System.Net;
+using System.Net.Sockets;
+using System.Text;
+using System.Threading;
+using System.Windows.Forms;
+using System.Threading.Tasks;
+
+namespace ElevatorSimulator
+{
+    public partial class Elevator : Form
+    {
+        private Lift _lift;
+
+        private Socket _serverSocket;
+        private Socket _clientSocket;
+        private byte[] _buffer;
+        public Elevator()
+        {
+            InitializeComponent();
+            InitializeDynamicComponent();
+            InitializeLift();
+            InitializeMethods();
+            CreateSarver();
+        }
+
+        void CreateSarver()
+        {
+            _serverSocket = new Socket(AddressFamily.InterNetwork, SocketType.Stream, ProtocolType.Tcp);
+            _serverSocket.Bind(new IPEndPoint(IPAddress.Parse("127.0.0.1"), 3333));
+            _serverSocket.Listen(10);
+
+            ConnectingSarver();
+        }
+
```

Comment:
private void Elevator_Load(object sender, EventArgs e) - doesn't do anything and is never called